Arc/Arc.js

```diff
@@ -0,0 +1,153 @@
+/**
+ * Agate styled arc components and behaviors.
+ *
+ * @example
+ * <Arc color="blue" endAngle={200} startAngle={0}" radius={100} />
+ *
+ * @module agate/Arc
+ * @exports Arc
+ * @exports ArcBase
+ * @exports ArcDecorator
+ */
+
+import kind from '@enact/core/kind';
+import Pure from '@enact/ui/internal/Pure';
+import ri from '@enact/ui/resolution';
+import PropTypes from 'prop-types';
+import compose from 'ramda/src/compose';
+import React from 'react';
+import Skinnable from '../Skinnable';
```

Comment:
Would you please add one empty line above this?

Reply:
sure